edenai_apis/apis/api4ai/api4ai_api.py

```
@@ -234,6 +234,69 @@ def image__face_detection(
         )
         return result
 
+    async def image__aface_detection(
+        self, file: str, file_url: str = "", **kwargs
```

Ruff (0.14.4)

Unused method argument: `file_url`

(ARG002)